packages/client/components/Facilitator.tsx

```diff
@@ -55,7 +55,7 @@ const Subtext = styled('div')({
   lineHeight: '16px'
 })
 
-const StyledIcon = styled(Icon)({
+const StyledIcon = styled(MoreVert)({
```

Comment:
-1 the icon cannot be styled in the same way, you might need to wrap it in a `div` or similar

Before:
![Facilitator button](https://user-images.githubusercontent.com/7331043/181715656-6592f1d5-7732-4e84-a940-87e68b622b81.gif)

Now:
![Facilitator button no outline](https://user-images.githubusercontent.com/7331043/181715693-c83911ea-5ed5-4b2e-8dc2-0f9d764ba4b1.gif)

Reply:
ok